static/sidebar.less

```diff
@@ -274,3 +277,19 @@ img.social-icon-small {
 			}
 		}
 }
+// Mobile sidebar By Bitovi logo
+.by-bitovi-container {
+	width: 140px;
+	height: 30px;
+	display: flex;
+	justify-content: center;
+	margin: 0 auto;
+}
+.bitovi.by-bitovi {
+	width: 100%;
+	height: 100%;
+	background: url(img/by-bitovi-logo.svg) no-repeat;
+	background-position: center;
+	background-size: contain;
+	color: rgba(0, 0, 0, 0);
+}
```

Comment:
Are all these styles duplicated now? Can we minimize what’s duplicated? Also, if there are any styles that aren’t doing anything, can we remove them?

<img width="613" alt="Screen Shot 2019-06-25 at 10 15 57 AM" src="https://user-images.githubusercontent.com/10070176/60119005-57e07e80-9732-11e9-8a05-3ecfd546dee8.png">